plugins/source/aws/docs/tables/aws_support_trusted_advisor_check_results.md

```diff
@@ -0,0 +1,25 @@
+# Table: aws_support_trusted_advisor_check_results
+
+https://docs.aws.amazon.com/awssupport/latest/APIReference/API_DescribeTrustedAdvisorCheckResult.html
+
+The primary key for this table is **check_id**.
+
+## Relations
+
+This table depends on [aws_support_trusted_advisor_checks](aws_support_trusted_advisor_checks.md).
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|account_id|String|
+|category_specific_summary|JSON|
+|check_id (PK)|String|
```

Comment:
same comment here (let me know if I got it wrong), but it looks like in addition to `check_id` we need `account_id`, `region` and `language_code`

Reply:
`account_id` and `region` yes, but `language_code` no as we don't get this for each language:
https://github.com/cloudquery/cloudquery/pull/8140/files#diff-e2281beb0f75f74e18cbf10164be3cb5c89e063090181e2493b16dc915937e64R32

Reply:
Yeah, but I would probably still include `language_code`, so it doesn't become a breaking change later (seeing as `language_code` is also part of the parent PK)